style/about-us.css

```diff
@@ -0,0 +1,104 @@
+.about-us > div {
+  flex-direction: column;
+  position: relative;
+}
+
+.who-we-are {
+  display: flex;
+  position: relative;
+  height: 130px;
+  margin-top: 40px;
+}
+.who-we-are .title {
+  width: 160px;
+  text-transform: capitalize;
+  border-right: 2px solid var(--supplementary-color-ui-01);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.who-we-are .title h2 {
+  margin-right: 20px;
+}
+.who-we-are .description {
+  width: 60%;
+  max-width: 750px;
+  margin-left: 20px;
+}
+.who-we-are .description p {
+  margin-inline: 20px;
+  font-size: 1.1rem;
+}
+.who-we-are article {
+  background-image: url("./../assets/images/others/fireHand.png");
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  background-size: 150px;
+  position: absolute;
+  z-index: 0;
+  top: -110px;
+  right: 0;
+  width: 24%;
+  max-width: 300px;
+  background-color: var(--primary-color-brand-03);
+  color: var(--text-inverse);
+}
+.who-we-are article p {
+  padding: 30px 25px;
+  margin: 0;
```

Comment:
each section in single file

Reply:
ok don't worry this is my bad, I though they were one section. thanks for the comment

Reply:
Done
![Screenshot 2024-05-28 113749](https://github.com/YAQ0T/XNGAGE-first-project-html-css/assets/63197328/b8442694-7b4b-47ad-a731-9d6d03d1d862)